company/strategy.md

```diff
@@ -56,6 +61,42 @@ Sourcegraph is the one place to find and fix things across all code.
+
+- Fast code search that scales to giant codebases and the universe of open-source code.
+- Precise code navigation that makes walking the reference graph as simple as point-and-click.
+- Make large scale refactorings possible (universal search-and-replace).
+- Integrate with code hosts and code review tools to bring these developer super powers to everywhere SWEs need to understand code.
+- Sourcegraph becomes a natural hub to distribute other dev tools, due to its extensibility and the fact that the average developer using Sourcegraph does so several times per day.
+- Sourcegraph provides analytics about what parts of the code are changing and makes this view accessible to every individual developer.
+
+### 5-year vision: democratize code {#5-year-vision}
+
+We will make the universal code graph accessible to everyone.
+
+The universe of code is exploding, but to any given person, most of that universe remains terra incognita. We will map out that universe and its graph of dependencies, references, and authorship, so that everyone can easily traverse, explore, discover, and make use of it. (Think Google vs. Yahoo.)
+
+-Sourcegraph understands the reference and dependency graph of the open-source universe and all our customers' private code. We make it easy to walk this graph (subject to the privacy constraints of our customers and users) even if you're not a professional software engineer.
+- You can see which changes to code affect you and how your changes affect others.
+- You can discover and more easily use libraries, with data about who uses them and how they're used.
+- Sourcegraph understands how the entire codebase of an organization is evolving and makes this accessible to codebase owners. The health and overall status of the codebase is no longer opaque to engineering leaders.
+
+### 10-year-vision: democratize coding {#10-year-vision}
+
+We will make it so coding is a universal skill (just like universal literacy). This requires not only making coding easier but also introducing more widely available economic incentives--letting anyone earn a living coding from anywhere in the world.
+
+As code becomes an essential part of every company and organization, more and more people can make an impact by having access to code. It will start with roles like product management and SRE that are adjacent to software engineering, but we envision a future where code becomes so critical that nearly everyone in an organization will have an interest in understanding it and modifying it. This applies to organizations of all sizes--and even individuals earning a living from libraries they wrote and shared with others who make profitable end-user products on top of it.
```

Comment:
```suggestion
As code becomes an essential part of every company and organization, more and more people can make an impact by having access to code. It will start with roles like product management and SRE that are adjacent to software engineering, but we envision a future where code becomes so critical that nearly everyone in an organization will have an interest in understanding it and modifying it. This applies to organizations of all sizes--and even individuals earning a living from libraries they wrote and shared with others who make profitable end-user products on top of them.
```